tests/cpp_validation_tests/test_validation.cpp

```diff
@@ -60,7 +60,7 @@ struct Buffer {
 void parse_single_object(RawDataPtr ptr, json const& j, MetaComponent const& meta, Idx position) {
     for (auto const& it : j.items()) {
         // Allow and skip unknown attributes
-        if (!meta.has_attribute(it.key())) {
+        if (static_cast<Idx>(meta.has_attribute(it.key())) == 0) {
```

Comment:
please also try to build locally with out the `static_cast`. i believe it should work just fine. if it doesn't then that's fine as well but please try it first

```suggestion
        if (meta.has_attribute(it.key()) == 0) {
```